File: Medium/Python/1769. Minimum Number of Operations to Move All Balls to Each Box.py
class Solution(object):
    def minOperations(self, boxes):
        # Initialize a list to contain the number of
        # operations
        ops = [0] * len(boxes)

        # Use a for loop to move all the balls to the ith box
        for i in range(len(boxes)):
            cost = 0
            for j in range(len(boxes)):
                if boxes[j] == '1':
                    cost += abs(i - j)
            ops[i] = cost

        # Return the ops list
        return ops

File: Medium/Java/1769. Minimum Number of Operations to Move All Balls to Each Box.java
class Solution {
    public int[] minOperations(String boxes) {
        // Initialize an array to contain the number of
        // operations
        int[] ops = new int[boxes.length()];

        // Use a for loop to move all the balls to the ith box
        for(int i = 0; i < boxes.length(); i++){
            int cost = 0;
            for(int j = 0; j < boxes.length(); j++){
                if(boxes.charAt(j) == '1')
                    cost += Math.abs(i - j);
            }
            ops[i] = cost;
        }

        // Return the ops array
        return ops;
    }
}
